### PR TITLE
Reenable IIS tests

### PR DESCRIPTION
**Pull request by Mpdreamz**
These were disabled because they started failing on CI possibly due to space constraints.

Re-enabling this again in a PR so that we don't forget this is not running

### CODE REVIEW

File: sample/Sample.Microsoft.Data.SqlClient/Sample.Microsoft.Data.SqlClient.csproj [hunk 1]
```diff
@@ -1,9 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net462;net6.0;net7.0</TargetFrameworks>
```
Comment (gregkalapos) [inline sample/Sample.Microsoft.Data.SqlClient/Sample.Microsoft.Data.SqlClient.csproj:4]: Is this change needed here? Mainly asking, because I remember SqlClient tracing works very differently between `net462` vs new .NET.

**Review by gregkalapos — COMMENTED** [review]
Nice! Just 1 small question.

File: sample/Sample.Microsoft.Data.SqlClient/Sample.Microsoft.Data.SqlClient.csproj (hunk 1 above)
Comment (Mpdreamz) [reply sample/Sample.Microsoft.Data.SqlClient/Sample.Microsoft.Data.SqlClient.csproj:4]: Its for a sample thats not used by testing, literally a playground I created a while back. I was even debating deleting it alltogether :)

`/sample` no longer holds applications in use by our tests, those have all moved to the `/test` folder.

**Review by gregkalapos — APPROVED** [review]
